manifests/init.pp

```diff
@@ -62,29 +130,6 @@
 # $plugin_version::             foreman plugins package version, it's passed to ensure parameter of package resource
 #                               can be set to 'installed', 'latest', 'present' only
 #
-# $db_manage::                  if enabled, will install and configure the database server on this host
-#                               type:boolean
-#
-# $db_type::                    Database 'production' type (valid types: mysql/postgresql/sqlite)
-#
-# $db_adapter::                 Database 'production' adapter
```

Comment:
Not sure that the unmanaged database options should be moved, I'd suggest only keeping `$db_type` in the main parameters docs and _possibly_ db_manage, as type is the main parameter for changing the DB impl.